app/views/enquiries/edit.html.erb

```diff
@@ -1,2 +1,2 @@
-<h1>Edit Enquiry</h1>
+<h1 class="jumbotron">Edit Enquiry</h1>
```

Comment:
I guess when you see this with bootstrap working, you'll see this is probably the wrong class here :)

Reply:
I take it your not a fan of jumbatrons....whats not to like....nice to make a big bold statement :)